packages/expo-notifications/src/NotificationsEmitter.ts

```diff
@@ -148,10 +159,23 @@ export async function getLastNotificationResponseAsync(): Promise<NotificationRe
  * @return A promise that resolves if the native call was successful.
  */
 export async function clearLastNotificationResponseAsync(): Promise<void> {
-  if (!NotificationsEmitterModule.clearLastNotificationResponseAsync) {
-    throw new UnavailabilityError('ExpoNotifications', 'getLastNotificationResponseAsync');
+  clearLastNotificationResponse();
+}
+
+/**
+ * Clears the notification response that was received most recently. May be used
+ * when an app selects a route based on the notification response, and it is undesirable
+ * to continue selecting the route after the response has already been handled.
+ *
+ * If a component is using the [`useLastNotificationResponse`](#uselastnotificationresponse) hook,
+ * this call will also clear the value returned by the hook.
+ *
+ */
+export function clearLastNotificationResponse(): void {
+  if (!NotificationsEmitterModule.clearLastNotificationResponse) {
+    throw new UnavailabilityError('ExpoNotifications', 'clearLastNotificationResponse');
   }
```

Comment:
Same comment here -- the code should check for the availability of `clearLastNotificationResponse`